src/app/shared/firecloud.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Dockstore } from './dockstore.model';
+
+@Injectable()
+export class FireCloudService {
+
+  private importRegEx: RegExp = new RegExp(/^\s*import\b/, 'm');
+
+  constructor() { }
+
+  /**
+   * Simple test to see if wdl has any import statements.
+   *
+   * Look for any statement starting with the word `import` as the first non-blank characters at the beginning
+   * of any line.
+   *
+   * @param {string} wdl
+   * @returns {boolean}
+   */
+  wdlHasImports(wdl: string): boolean {
```

Comment:
This brings up a good question. 
How well is Dockstore dealing with `import` as a keyword anyway?
Do we properly create secondary files, if so this could check for >0 secondary files rather than regex although this works for now

Reply:
http imports should work though, shouldn't they? Although FC asked us to disable for even http import for now.

Reply:
Yes, http imports should work no matter what. 
I was just curious what the state of relative imports was. But you're right, http imports would have to be looked for like this either way.